plugin/docker/container.go

```diff
@@ -15,35 +14,43 @@ import (
 
 type container struct {
 	plugin.EntryBase
-	client    *client.Client
-	startTime time.Time
+	client *client.Client
 }
 
-// Metadata
-func (c *container) Metadata(ctx context.Context) (plugin.MetadataMap, error) {
+func (c *container) Metadata(ctx context.Context) (plugin.EntryMetadata, error) {
 	// Use raw to also get the container size.
 	_, raw, err := c.client.ContainerInspectWithRaw(ctx, c.Name(), true)
 	if err != nil {
 		return nil, err
 	}
 
-	return plugin.ToMetadata(raw), nil
-}
-
-// Attr
-func (c *container) Attr(ctx context.Context) (plugin.Attributes, error) {
-	return plugin.Attributes{
-		Ctime: c.startTime,
-		Mtime: c.startTime,
-		Atime: c.startTime,
-	}, nil
+	return plugin.ToMeta(raw), nil
 }
 
 func (c *container) List(ctx context.Context) ([]plugin.Entry, error) {
+	// TODO: May be worth creating a helper that makes it easy to create
```

Comment:
Yes, I think it'd make it more readable. The different behavior for `meta["Size"]` is kind of annoying though. How does the `Sync` behavior handle it?

Reply:
OK, I can add a `plugin.NewMetadataJSONFile` helper.

The `Sync` behavior handles it by munging the sizes -- see https://github.com/ekinanp/wash/blob/common-metadata/plugin/syncableAttribute.go#L147-L170

Reply:
Actually, let me add the helper once we see more use-cases some up, and when the `metadata.json` files are something that people like. I don't think keeping it the way it is is that bad right now.